testsuite/tests/warnings/w59.ml

```diff
@@ -14,6 +14,7 @@ compile_only = "true"
 
 * flambda
 compiler_reference = "${test_source_directory}/w59.flambda.reference"
+flags = "-w +A-70 -dflambda-invariants"
```

Comment:
Note: we now have a technique to avoid having to change reference line numbers when we edit the test block, which is to use the TEST_BELOW feature to have the test block at the end (see backtrace/backtrace.ml). Of course if you decide to move to that you have to change the line numbers anyway, but hopefully for the last time.

Reply:
Ah, I didn't think about that. Do you think it would be a good idea to make a big PR changing all tests that check line numbers at once ? Or just switch individual tests when we need to fix line numbers ?

Reply:
I'd go the incremental route, and focus only on tests whose reference output contains line numbers. You had to touch a good number of those in your fixing-flambda work, maybe you want to change them all those at once.

Reply:
Well, all tests involving backtraces or compiler warnings will likely be concerned, so a global sweep of the corresponding directories would make sense too. The incremental approach still makes sense for the rest of the tests.